src/topology/sheaves/structure_sheaf.lean

```diff
@@ -0,0 +1,214 @@
+/-
+Copyright (c) 2020 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin, Scott Morrison
+-/
+import algebraic_geometry.prime_spectrum
+import ring_theory.localization
+import algebra.category.CommRing
+import topology.sheaves.local_predicate
+import topology.sheaves.forget
+import ring_theory.bundled_subring
+
+/-!
+# The structure sheaf on `prime_spectrum R`.
+
+-/
+
+universe u
+
+noncomputable theory
+
```

Comment:
All of this file should be namespaced.
How about `algebraic_geometry.spec`?

Reply:
I moved the whole file to `src/algebraic_geometry`, and will namespace.